docs/fundamentals/syslib-diagnostics/syslib1021.md

```diff
@@ -0,0 +1,15 @@
+---
+title: SYSLIB1021 error
+description: Learn about the diagnostic that generates compile-time error SYSLIB1021.
+ms.date: 05/07/2021
+---
+
+# SYSLIB1021: Can't have the same template placeholders with different casing
```

Comment:
```suggestion
# SYSLIB1021: Can't have the same template item names with different casing
```

I borrowed the word "item" from what things like `{0}` are referred to in String.Format strings. What do you think?